cfme/utils/template/virtualcenter.py

```diff
@@ -18,8 +18,10 @@ class VMWareTemplateUpload(ProviderTemplateUpload):
     def upload_template(self):
         cmd_args = [
             "ovftool --noSSLVerify",
-            # prefer the datastore from template_upload
-            "--datastore={}".format(self.provider_data.provisioning.datastore),  # move later
+            # Deploy on the datastore for provision, move to template datastore later
+            # JUWATTS CHANGE THIS
+            "--datastore={}".format(self.provider_data.provisioning.datastore_cluster),
```

Comment:
I would think we would want to check for a few fields here in provisioning - `datastore_cluster`, `datastore`, or `allowed_datastores`?

Reply:
This is setup in #8609 , ignore this.